.rubocop.yml

```diff
@@ -1 +1,19 @@
 inherit_from: .rubocop_todo.yml
+AllCops:
+  Exclude:
+    - 'test/dummy/**/*'
+    - 'tmp/**/*'
+    - 'spec/fixtures/**/*'
+    - 'vendor/**/*'
+  TargetRubyVersion: 2.3
+
+# Apparently a thing in Rubocop for Ruby 2.3+ in prep for Ruby 3
```

Comment:
Ruby 2.3 has a feature where you can signal that it should behave like (unreleased) ruby 3.0 with regard to string literals. You do this by putting a comment at the top of the file (below the `#!` if applicable):

```ruby
# frozen_string_literal: true
```
I'm pretty sure rubocop can auto-add this for you.

Good writeup [here](https://wyeworks.com/blog/2015/12/1/immutable-strings-in-ruby-2-dot-3)